multi/imxrt-multi/config.h

```diff
@@ -766,6 +766,64 @@
 #endif
 #endif
 
+#if defined(UART_CONSOLE)
+#if ISEMPTY(UART_CONSOLE)
```

Comment:
Actually, there are no restrictions for this - `UART_CONSOLE` may be empty, even though it may be handled differently in different modules (plo, kernel).

Reply:
OK, I removed this check.